ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -222,11 +223,28 @@ private[commands] class CommandTracker[Context](
             s"A command with id $commandId is already being tracked. CommandIds submitted to the CommandTracker must be unique."
           ) with NoStackTrace
         }
-        val commandTimeout =
-          timeoutFromDeduplicationPeriod(commands.deduplicationPeriod, maximumCommandTimeout)
+        val commandTimeout = submission.value.timeout match {
```

Comment:
This is really a duration so should be named accordingly (e.g., to avoid confusion with the corresponding parameter in `TrackingData`).

Reply:
Well, we use "timeout" everywhere else. I'm happy to change it but let's do it in a separate PR, after proper discussion.

I don't think `commandDuration` really captures the meaning. I went for `commandExpiry` before but that's also confusing.